### DISCUSSION TITLE
The starlette must be upgraded to 0.21.0 to solve this bug:https://github.com/tiangolo/fastapi/issues/5403

### DISCUSSION THREAD

**Discussion by caishengxiang** [fastapi/fastapi#8397] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
fastapi>=0.85.1
starlette>=0.21.0
```

### Description

these package versions have conflicting dependencies.
I want to use starlette=0.21.0.

### Operating System

Linux

### FastAPI Version

0.85.1

### Python Version

python--3.9.7

**Comment by JarroVGIT** [comment, marked as answer]
There is already a PR for that, no need for another issue. #5471

**Comment by raphaelauv** [comment]
@caishengxiang could please close the issue -> https://github.com/tiangolo/fastapi/releases/tag/0.87.0
